qemu.go

```diff
@@ -313,7 +313,7 @@ func (q *qemu) appendSocket(devices []ciaoQemu.Device, socket Socket) []ciaoQemu
 func networkModelToQemuType(model NetInterworkingModel) ciaoQemu.NetDeviceType {
 	switch model {
 	case ModelBridged:
-		return ciaoQemu.TAP
+		return ciaoQemu.MACVTAP
```

Comment:
Why did you change this ?

Reply:
@sboeuf The mq link uses the same connection methodology as macvtap. Hence the change. Longer term I should change the name of the ciaoQEMU type should be ciaoQemu.FD. macvtap was too specific as it is just a fd based networking connection.

I will rename/rework the qemu networking code quite a bit when adding support for vhost-user and macvlan and tc based connectivity. Will do the renaming as part of that just to avoid revendoring qemu.